chalice/cli/__init__.py

```diff
@@ -499,22 +499,30 @@ def generate_models(ctx, stage):
                     "this argument is specified, a single "
                     "zip file will be created instead. CloudFormation Only."))
 @click.option('--merge-template',
-              help=('Specify a JSON template to be merged '
+              help=('Specify a JSON or YAML template to be merged '
                     'into the generated template. This is useful '
                     'for adding resources to a Chalice template or '
                     'modify values in the template. CloudFormation Only.'))
+@click.option('--template-format', default='json',
```

Comment:
default is `json`, but the doc string only mentions `JSON` and `YAML`. Later on in the code it explicitly checks for `yaml` lower case. I think if you use it as documented, you will silently get JSON instead. Given that yaml and json are hardcoded later on in different modules, it should probably be normalized to lower case here.

Reply:
Feedback incorporated.